package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "pay": "ts-node index.ts",
+    "pay": "ts-node index.ts pay",
```

Comment:
Trying to run this locally as `npm run pay --cycle=489 --dry-run` but it's no longer recognizing the arguments. Does it have to be run differently?

Reply:
Yea it should be now `npm run pay -- --dry-run 489`